index_node/solr.py

```diff
@@ -89,15 +85,16 @@ def start_solr(solr_config_type, port_number, SOLR_INSTALL_DIR="/usr/local/solr"
     start_solr_options = ["start", "-d", server_directory, "-s", solr_solr_home, "-p", port_number, "-a", enable_nodes, "-m", "512m"]
     esg_functions.call_binary("/usr/local/solr/bin/solr", start_solr_options)
 
-    solr_status(SOLR_INSTALL_DIR)
-
-def solr_status(SOLR_INSTALL_DIR):
+def solr_status():
     '''Check the status of solr'''
     try:
-        esg_functions.call_binary("/usr/local/solr/bin/solr", ["status"])
-    except ProcessExecutionError:
-        logger.error("Error checking solr status")
+        result = esg_functions.call_binary("/usr/local/solr/bin/solr", ["status"])
```

Comment:
Can probably remove the use of the result variable here since we are just returning a boolean